testing/scripts/runtests

```diff
@@ -1,5 +1,13 @@
 #!/bin/sh -u
+export PATH=$PATH:~/.local/bin
 export CLASSPATH=.:$(pwd)/data/javatools.jar
-mkdir -p dist/test
-cabal build
-cabal exec -- cabal run -- --xml=dist/test/bnfc-system-tests.xml $*
+cd ../source
+stack setup
+stack init
+stack test
+stack install
```

Comment:
Just to clarify: in the branch I linked up there the system tests can be run in docker and locally. So the docker only adds a new possibility to run all the tests in a well configured environment. (But this can be discussed later on in another issue)